### ISSUE TITLE
Dynamic Scroll View Content

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#1]
<sup>Issue created by Tim Day as [Bitbucket Issue #&#8203;1](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/1) on 2014.12.31 01:24.</sup>
From Reddit:

Adding content to scroll views at run time is currently a pain in the ass. If you have a list of prefabs you want to add into or remove from a scroll view it is a real pain. Scroll views base their movement/clipping off the size of the scroll view container. Not the contents of the scroll view/ So you have to adjust the size of the scroll view to fit all the children. There's a bit of functionality built in to do this but it doesn't work at run time.

A function that takes a UI rect and make it encompass the entire size/area of its children would be a good start.

tl;dr scroll views aren't content driven in uGUI which makes dynamic/run time scroll view content tough.
